doc/man/scons.xml

```diff
@@ -4925,6 +4925,13 @@ is not being used.</para>
   </listitem>
   </varlistentry>
 
+  <varlistentry>
+  <term><replaceable>n</replaceable>.<varname>relpath</varname></term>
```

Comment:
I guess I'd have preferred the term `toprel` here - releative to the Top (aka SConstruct-containing) dir, as that term has started to creep in elsewhere (when describing `#path/to/something` form).  But if @bdbaddog as maintainer is okay with it, then I don't consider my opinion a blocker to the PR.

Reply:
I have no objections to the rename, it is more self-documenting, and would leave the possibility of a relpath that isn’t necessarily relative to the top SConstruct.

Reply:
.relpath is better IMHO.. (maps to python API).